### DISCUSSION TITLE
2024/02/17/cloud-storage-cross-platform-mount-and-access/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#15] [General]
# 2024/02/17/cloud-storage-cross-platform-mount-and-access/

刚放寒假那几天，有位朋友问了我一个问题： 有什么不限制文件格式的、基于网络的文件互传/同步工具或云储存，以便于代码/文档/图片等在不同设备之间的共享？ 这个问题之前没有仔细研究过，模模糊糊也只是知道有一些技术可以实现这样的文件共享，于是想着专门学习一下。然而，寒假期间除了休息和给长辈帮忙，就没怎么给学习留下时间。这不，趁着寒假快要结束，刚好抽出点时间来探索一下这个问题。

https://wz.anoms.top/2024/02/17/cloud-storage-cross-platform-mount-and-access/

**Comment by Hazelandinstant** [comment]
我最近找到了适合自己的办法：实验室有一个服务器，是可以通过web方式打开的，一般用于存储实验室大家的code，所以容量不大。
但是好处就是没有网络限制，在哪都能打开，上传和下载速度都非常可以，还能连接实验室其他的计算用的服务器。
果然我还是太年轻了，原来实验室早有安排！

**Reply by cyclinbox** [reply]
👍真好hhhh，看来同步代码的问题已经解决了，而且从描述中感觉这个服务器非常强大（有点羡慕~）
不过，“实验室有一个服务器，一般用于存储实验室大家的code”，这让我想到了GitLab和SVN。好奇你们用的是这些工具吗？

**Reply by Hazelandinstant** [reply]
以我浅薄的知识来看，我觉得似乎都不是。
台湾群晖科技（这是我能找到的有限线索）

**Reply by cyclinbox** [reply]
！！！
“群晖科技”，这不会是群晖系统的NAS吧！

**Reply by Hazelandinstant** [reply]
是的！确实如此！
